class-wc-facebookcommerce.php

```diff
@@ -185,6 +188,10 @@ public function init() {
 				$this->connection_handler = new \SkyVerge\WooCommerce\Facebook\Handlers\Connection( $this );
 				$this->webhook_handler = new \SkyVerge\WooCommerce\Facebook\Handlers\WebHook( $this );
 
+				if ( \WC_Facebookcommerce_Utils::isWoocommerceIntegration() ) {
+					$this->tracker = new \SkyVerge\WooCommerce\Facebook\Utilities\Tracker();
```

Comment:
I've added this class to `Utility` folder and followed the existing namespace pattern (`SkyVerge`). Keen for feedback on all this and also how the code is loaded.

Reply:
I would love to start using just the `WooCommerce` namespace without `SkyVerge` in the front since we are doing this now.

Reply:
But that is something that we should address in a separate PR with a bigger scope.

Reply:
> start using just the WooCommerce namespace without SkyVerge 

Can you add an issue for that? Or just add to retro board. It's bugging me too :)